frontend/src/types.ts

```diff
@@ -58,6 +58,22 @@ export const nip47MethodDescriptions: Record<RequestMethodType, string> = {
   [NIP_47_PAY_INVOICE_METHOD]: "Send payments",
 };
 
+export const expiryOptions: Record<string, number> = {
+  "1 week": 7,
+  "1 month": 30,
+  "1 year": 365,
+  Never: 0,
+};
+
+export const budgetOptions: Record<string, number> = {
+  "10k": 10_000,
+  "25k": 25_000,
+  "50k": 50_000,
+  "100k": 100_000,
+  "1M": 100_000_000,
```

Comment:
Was this a mistake, and since when?

```suggestion
  "1M": 1_000_000,
```

Reply:
Oh shit

Reply:
Luckily there aren't many (ig the ones created after November are wrongly set: https://metabase.getalby.com/question#eyJkYXRhc2V0X3F1ZXJ5Ijp7InR5cGUiOiJxdWVyeSIsInF1ZXJ5Ijp7InNvdXJjZS10YWJsZSI6NjQsImZpbHRlciI6WyI9IixbImZpZWxkIiwzNzUsbnVsbF0sMTAwMDAwMDAwXX0sImRhdGFiYXNlIjo0fSwiZGlzcGxheSI6InRhYmxlIiwidmlzdWFsaXphdGlvbl9zZXR0aW5ncyI6e319) should we change those?

Reply:
I don't think it's an issue in the original repo. Just we need to fix it here :pray: